.gitlab/newell.gitlab-ci.yml

```diff
@@ -0,0 +1,54 @@
+
+cleanup:
+ stage: .pre
+ extends:
+  - .SVC-Account-Cleanup
+
+hykkt-test-newell:
+  stage: test
+  needs: []
+  variables:
+    SLURM_Q: "newell_shared"
+    MY_CLUSTER: "newell"
+  extends:
+  - .pnnl_tags_template
+  - .hykkt_script_template
+
+
+.report-status:
+  image: mrnonz/alpine-git-curl:alpine3.16
+  variables:
+    GIT_STRATEGY: none
+    STATUS_PROJECT: ORNL/hykkt
+    STATUS_NAME: CentOS-8-ppc64le-gcc/8.5.0-cuda-11.4-v100-arch/70
+
+  extends:
+  - .pnnl_nonhpc_tags
+  script:
+    - |
+      set -x
+      curl -L \
+      -X POST \
+      -H @${GITHUB_CURL_HEADER}\
+       https://api.github.com/repos/${STATUS_PROJECT}/statuses/${CI_COMMIT_SHA} \
+       -d "{\"state\":\"${CI_JOB_NAME}\",\"target_url\":\"${CI_PIPELINE_URL}\",\"context\":\"${STATUS_NAME}\"}"
```

Comment:
Since we escape everything with quotes nicely here, the CI job names with spaces should work...